src/views/ConsentContinuationOut.vue

```diff
@@ -321,10 +321,11 @@ export default class ConsentContinuationOut extends Mixins(CommonMixin, DateMixi
   @Getter(useBusinessStore) getLegalName!: string
   @Getter(useConfigurationStore) getPayApiUrl!: string
   @Getter(useRootStore) getUserInfo!: any
-  @Getter(useRootStore) isRoleStaff!: boolean
 
   // enum for template
   readonly FilingCodes = FilingCodes
+  readonly AuthorizedActions = AuthorizedActions
+  readonly IsAuthorized = IsAuthorized
 
   // variables for Certify component
   certifiedBy = ''
```

Comment:
Should there be a check in created(), below, that the current filing type is authorized? I'm thinking yes, just to prevent smart users from trying to go straight to that route (thus bypassing the dashboard).

Reply:
I see you did this in some filings. I think this applies to all filings.